docs/en/setup/Configuration-File.md

```diff
@@ -159,6 +159,7 @@ verify:
   retry:            # verify with retry strategy
     count: 10       # max retry count
     interval: 10s   # the interval between two attempts, e.g. 10s, 1m.
+  fail-fast: false  # when a case fails, whether to stop verifying other cases. This property defaults to false.
```

Comment:
What about setting the default value to `true` so it behaves just like before?

Reply:
OK! Delayed a little time because of lunch. I will fix as soon as possible!

Reply:
> OK! Delayed a little time because of lunch. I will fix as soon as possible!

Please don't be stressful, take your time 🍖

Reply:
😊

Reply:
Looks like you just modify the value in example config file and doc, but in codes the default value is still `false`. It seems there is no easy way to set a default value for `bool` in struct. @fgksgf any idea?

Reply:
How about setting the default `true` here https://github.com/apache/skywalking-infra-e2e/blob/82494d053f879c2e10bc6c1439c05673d50bb37b/internal/config/globalConfig.go#L41

Reply:
@chunriyeqiongsaigao please read this 👆

Reply:
OK, I will write the init() function!